### ISSUE TITLE
Add UA button

### ISSUE THREAD

**Issue by Hjard** [pumpi/troop-creator#99]
I would recommend an "add UA button" in the list view. A simple plus button that adds the respective UA to the unit. 
This would make handling easier especially on smaller, handheld devices.

Possible problems: 
Mage Hunter Strike Force has 2 UAs available.
Mercenary Units can have either their UA, a Ranking Officer (when played in another army) or a Cephalyx Dominator (in the Operation Theater theme force).

**Comment by pumpi** [comment]
i have wrote an function that can check if an model can join the group for the journeyman function this can do his job for this enhancement an check if more then one UA ;)

**Comment by SirMythos** [comment]
Maybe a solution similar to the Warjack-List with the Journeyman?

**Comment by pumpi** [comment]
yes i think so. I whil thest some at the next days then we can add this enhancement
